_posts/2018-11-24-artsy.md

```diff
@@ -0,0 +1,506 @@
+
+Even knowing all this, I still bought all my stock when I left. I struggle to see Artsy failing, but maybe not to the
+same level of potential it used to have.
+
+### Mismanaged Introduction of a Product Org
+
+Artsy got real far with just design and engineering. A typical tech company would have three sets of roles for the
+building of user-facing products: Design, Product and Engineering. At a gross simplification: What it look and feel like
+(design), what it should do (product) and how should it work (engineering.)
+
+Artsy's older organization structure stems from the perspective of [Robert Lenne][rl], the company's original head of
+product and design. He built a culture of product design within the design team, effectively giving the designers in
+Artsy the responsibilities of what should be built within the company. This worked very well when the company was
+smaller, but the concept was starting to stretch at the seams as design and development doubled in size.
+
+The answer from leadership was that we needed to split those responsibilities and become a more traditional organization
+structure. I agreed with this idea, we'd had great experiences in the past where someone had taken up a product
+management role within the auctions team.
+
+The first six months of changing the team's structure were pretty disastrous. For the first time, people who I had hired
+left the company and the engineering culture took a hit which it never really recovered from. It felt like quite a lot
+of new people were under a lot of pressure to show results very quickly from these team changes. This pressure created
+incentives for people to take shortcuts, or make culture changes hard and fast without necessarily covering all their
+bases. People in all aspects of the business felt a loss of control and we spent a long time trying to find ways to get
+that feeling back. Gradually, it just kinda re-appeared with time and effort.
+
+Obviously, I don't think there's a world without pressure of some sort - but we'd had many successful and considered
+re-orgs within the company over the last 8 years. Every product person and most dev leads involved in figuring out what
+that transition should look like (myself including) aren't with the company anymore. Looking back, I keep thinking there
+a few places where maybe I could have stepped up more, and I regret I didn't. I even expressed worries about it on two
+years ago in [On Being 31][31]
+
+It's worth stressing that Artsy engineering was in **a solid place** as I left, this transition happened close to two
+years ago and I wouldn't have left if it wasn't going to be OK. The folks working at Artsy today are fun, interesting
+and making the art world a better place. If you're considering a job there and somehow found this first,you shouldn't
+consider this to be a blocker. The company is different, and stable now - it was just time for me and 2 years of shaky
+foundations was enough.
```

Comment:
Not gonna lie, this was a difficult section to read for me. Especially as Artsy continues to change under the new leadership – some change are super-exciting and others have been disruptive. The stability I desire still seems on the horizon, but it feels like it's getting closer. 

I can't find fault with anything you've said here.

If you want, you could mention some of the work others did to help address the culture: https://artsy.github.io/blog/2019/04/19/having-a-coffee-with-every-engineer/